examples/website-demos/virtual-event/src/components/Chat/ChatContainer.tsx

```diff
@@ -19,6 +19,7 @@ import { GiphyContextProvider } from '../../contexts/GiphyContext';
 import { useInitChat } from '../../hooks/useInitChat';
 
 import { Channel as StreamChannel, UserResponse } from 'stream-chat';
+import { ThreadHeaderUI } from './ThreadHeaderUI';
```

Comment:
move this up with the other component imports and let's just call it ThreadHeader, only added UI on Message and MessageInput because i think they need a little extra semantics